__tests__/ExpensiMark-Markdown-test.js

```diff
@@ -373,6 +373,19 @@ test('Test anchor tags replacements', () => {
 
     expect(parser.htmlToMarkdown(urlTestStartString)).toBe(urlTestReplacedString);
 });
+
+test('Test link markdown with uppercase letter in https replacement', () => {
+    const testString = '[Https://www.regex101.com](Https://www.regex101.com)';
+    const resultString = '<a href=\"Https://www.regex101.com\" target=\"_blank\" rel=\"noreferrer noopener\">Https://www.regex101.com</a>';
+    expect(parser.replace(testString)).toBe(resultString);
+});
+
+test('Test autolink with uppercase letter in https replacement', () => {
+    const testString = 'Https://www.regex101.com';
+    const resultString = '<a href=\"Https://www.regex101.com\" target=\"_blank\" rel=\"noreferrer noopener\">Https://www.regex101.com</a>';
+    expect(parser.replace(testString)).toBe(resultString);
+});
```

Comment:
To double check that the opposite operation is not broken, can we add a test of html to markdown where the html to converts contains a link like `<a href=\"Https://www.regex101.com\" target=\"_blank\" rel=\"noreferrer noopener\">Https://www.regex101.com</a>`

🙏

Reply:
Something like

```js
test('Test link with uppercase href', () => {
    const testString = '<a href="Https://google.com/">multiline\ntext</a>';
    const resultString = '[multiline\ntext](Https://google.com/)';
    expect(parser.htmlToMarkdown(testString)).toBe(resultString);
});
```

in [__tests__/ExpensiMark-Markdown-test.js](https://github.com/Expensify/expensify-common/blob/9cf047b9741d3c02820d515dccb9e0a45b6595f5/__tests__/ExpensiMark-Markdown-test.js#L576)

Reply:
Oh, I see and added the opposite operation test

Reply:
Oh, I just included the test into the bundle test of anchor tags replacement. Is it good?

Reply:
Yes, this is fine.